public/assets_old/css/admin.css

```diff
@@ -0,0 +1,159 @@
+* {
+	margin: 0;
+	padding: 0;
+}
+body {
+	font-family: Arial,sans-serif;
+	font-size: 16px;
+	color:#333;
+	background:#FEFEFE;
+	min-width:320px;
+}
+input {
+	font-size: 1.125em;
+	padding: .125em .25em;
+	border: 1px solid #6A6A6A;
```

Comment:
Codacy found an issue: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/manual/common/buryak/pullRequest?prid=5800482)